### ISSUE TITLE
Route stats not selectable

### ISSUE THREAD

**Issue by matozoid** [theCrag/tca#38]
When looking at the route stats graph, I am tempted to click on a bar (like 4a-6a) to find the routes in that grade to see what I can climb. That does not do antyhing.

**Comment by bkucsera** [comment]
Great idea! We can put it up to the development list.
It needs some further development :) What should happen when clicking on a bar? For example we can drop the user to the route list, automatically filtered to 4a-6a.
Or you have other idea?

**Comment by matozoid** [comment]
Yeah, the filter seems to be the most obvious thing to do. It seems to fit with what I would do at a crag: "alright, I've arrived, which areas have routes that I can try?" "Okay, nice sector, show me the routes around grade 5".

This could tie in with the stats that thecrag keeps on climbers - it knows what level you're supposed to climb at already.

**Comment by bkucsera** [comment]
We will do this, thanks for the idea! Maybe in the next release, not the first one, but its on the list.
